cpu/stm32/periph/spi.c

```diff
@@ -64,6 +68,23 @@ static inline bool _use_dma(const spi_conf_t *conf)
 }
 #endif
 
+static uint8_t _get_clkdiv(const spi_conf_t *conf, uint32_t clock)
+{
+    uint32_t bus_clock = periph_apb_clk(conf->apbbus);
+    uint8_t div = bus_clock / (2 * clock);
+    if (div > 1) {
```

Comment:
should be `if (div > 0)` : else if clock = bus_clock / 2 then div = 1 and the function return 1 insteed of 0.

Reply:
This should be resolved with the current version